### PR TITLE
Limited and restricted plugin test with current app

### PR DESCRIPTION
**Pull request by volodymyrss**
test plugins if `test-plugins` PR label is set. This is one of the possible ways. Let's not set the label unless and until we mean it.

### CODE REVIEW

File: .github/workflows/python-plugins.yml [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip flake8 pytest mypy pylint pytest-cov    
+        
+        if [ -f plugin/requirements.txt ]; then 
+          < plugin/requirements.txt grep -v oda-hub/dispatcher-app | grep -v cdci-data-analysis | grep -v oda_api > plugin-req.txt
+          pip install -r plugin-req.txt ./plugin[test] .[test]; 
+        fi
```
Comment (burnout87) [inline .github/workflows/python-plugins.yml:62]: does this mean that the `plugin/requirements.txt` file will have to be included? Or is it created from one of the previous step? I am not familiar yet with github workflows

**Review by burnout87 — APPROVED** [review]
just one question

File: .github/workflows/python-plugins.yml (hunk 1 above)
Comment (volodymyrss) [reply .github/workflows/python-plugins.yml:62]: yes, it is created in the other step. Plugin is cloned into `plugin` directory. This works.

Some tests fail for `integral` plugin. Strange. I look.

Comment (volodymyrss) [reply .github/workflows/python-plugins.yml:62]: That's unrelated. But strange.

**Pull request merged**